File: src/main/java/me/nasukhov/TukitaLearner/bot/io/ChannelRepository.java
package me.nasukhov.TukitaLearner.bot.io;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import java.util.Optional;

@Repository
public interface ChannelRepository extends JpaRepository<Channel, String> {
    Optional<Channel> findById(String id);
}

File: src/main/kotlin/me/nasukhov/tukitalearner/bot/io/ChannelRepository.kt
package me.nasukhov.tukitalearner.bot.io

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface ChannelRepository : JpaRepository<Channel, String> {
    override fun findById(id: String): Optional<Channel>
}
